app/views/content/life-as-a-teacher/teaching-as-a-career/change-careers/_article.html.erb

```diff
@@ -0,0 +1,111 @@
+    <p><a href="/train-to-be-a-teacher/if-you-have-a-degree"> Find out more about postgraduate teacher training</a>.</p>
+    <h3>If you do not have a degree</h3>
+    <p>If you do not have a degree, you can:</p>
+    <ul>
+      <li>do undergraduate teacher training to get a degree alongside qualified teacher status QTS</li>
+      <li>check if you're eligible to do a teaching degree apprenticeship</li>
+    </ul>
+    <p>Find out more about <a href="/train-to-be-a-teacher/if-you-dont-have-a-degree">training to teach if you do not have a degree</a>.</p>
+    <h3>If you're unable to get a degree</h3>
+    <p>You do not need a degree and QTS to <a href="/life-as-a-teacher/age-groups-and-specialisms/further-education-teachers">teach in further education</a> or to <a href="/life-as-a-teacher/age-groups-and-specialisms/early-years-teachers">teach in early years</a>. So if you want to become a teacher, but are not able to get a degree, you can explore these options.</p>
+  </section>
+  <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+    text: "I would advise anyone thinking about changing careers that you will be embraced and more importantly an asset to the profession and students.",
+    name: "Ian, career changer",
+    large: false
+  ) %>
+  </section>
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Fund your training</h2>
+    <p>Tax-free bursaries and scholarships of upto <%= v :scholarships_generic_maxshortened %> are available for eligible trainee teachers. You can get these alongside a tuition fee loan and maintenance loan.</p>
+    <p>You may also be able to get extra funding and support if you're:</p>
+    <ul>
+      <li>a parent or carer</li>
+      <li>disabled</li>
+      <li>a veteran</li>
+    </ul>
+    <p>Find out how you can <a href="/funding-and-support">fund your teacher training</a>.</p>
+  </section>
+</div>
+<div class="row">
+  <div class="col-space-l col-space-l-top">
+    <%= render Content::QuoteWithImageComponent.new(
+      title: "Teacher stories - Claire",
+      text: "<p>As a parent and professional with a regular income, the financial support available to me as a trainee physics teacher made the option of re-training seem not only viable, but attractive.</p><a href='/life-as-a-teacher/teaching-as-a-career/becoming-a-parent-sparked-my-interest-in-teaching'>Read more about Claire's career change to teaching</a>",
+      quotes: true,
+      background_color: "blue",
+      heading_color: "pink",
+    ) %>
+  </div>
+</div>
+<div class="row inset">
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Support for career changers</h2>
+    <h3>Finding and applying for training</h3>
+    <p>Our experienced teaching advisers can support you through the switch if you're thinking about changing careers. You can chat to an adviser by phone, text or email.</p>
```

Comment:
```suggestion
    <p>Our experienced teacher training advisers can support you through the switch if you're thinking about changing careers. You can chat to an adviser by phone, text or email.</p>
```

Reply:
I think we could bump this up a bit - eg along the lines of "they support hundreds/thousands (?) of career changers like you each year" (check with Candidate support)